models/project/project.go

```diff
@@ -227,15 +227,15 @@ func (u *ProjectStruct) NoneList() *list.ListStruct {
 	err := table.QueryRow("select id, project_id, user_id, title, color, list_option_id from lists where project_id = ? and title = ?;", u.ID, config.Element("init_list").(map[interface{}]interface{})["none"].(string)).Scan(&id, &projectID, &userID, &title, &color, &optionID)
 	if err != nil {
 		// noneが存在しないということはProjectsController#Createがうまく行ってないので，そっちでエラーハンドリングしてほしい
-		logging.SharedInstance().MethodInfo("Project", "NoneList", true).Panic(err)
+		panic(err)
 	}
 	if projectID == u.ID && title.Valid {
-		return list.NewList(id, projectID, userID, title.String, color.String, optionID)
+		return list.NewList(id, projectID, userID, title.String, color.String, optionID), nil
 	}
-	return nil
+	return nil, errors.New("none list not found")
 }
 
-func (u *ProjectStruct) Repository() *repository.RepositoryStruct {
+func (u *ProjectStruct) Repository() (*repository.RepositoryStruct, error) {
```

Comment:
__[golint]__
* [golint] exported method ProjectStruct.Repository should have comment or be unexported